client/my-sites/themes/theme-download-card/index.jsx

```diff
@@ -0,0 +1,40 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import i18n from 'lib/mixins/i18n';
+
+var ThemeDownloadCard = React.createClass( {
+
+	propTypes: {
+		theme: React.PropTypes.string.isRequired,
+		href: React.PropTypes.string
+	},
+
+	render() {
+		var downloadURI = this.props.href || ( 'https://downloads.wordpress.org/theme/' + this.props.theme + '.zip' );
```

Comment:
Let's add here an inline code comment explaining why we do fallback by default to ".org". ;)

Like:

```// When we don't generate zips, it's because we have released the theme on .org.```

Maybe?